near-sdk/src/utils/stable_map.rs

```diff
@@ -41,6 +42,7 @@ impl<K, V> StableMap<K, V> {
     pub(crate) fn inner(&mut self) -> &mut BTreeMap<K, Box<V>> {
         self.map.get_mut()
     }
+    #[cfg(feature = "unstable")]
     pub(crate) fn map_value_ref<Q: ?Sized, F, T>(&self, k: &Q, f: F) -> Option<T>
```

Comment:
hmm, how come this function is `unstable`?

Reply:
because the collections that use it aren't stabilized in this PR. Otherwise this would warn for being unused